src/dialogs/tododialog.ui

```diff
@@ -105,16 +105,19 @@
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
+            <property name="tickPosition">
```

Comment:
What are those new 3 lines for?

Reply:
This adds the tickmarks to the priority slider so it is easier to distinguish between priority and progress slider by glance. Check the screenshot in PR summary for what I mean :)